src/common/css/style.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: "Roboto", "Noto Sans JP", serif;

```

Comment:
```suggestion
  font-family: "Roboto", "Noto Sans JP", sans-serif;
```

![image](https://user-images.githubusercontent.com/43571743/131622458-7f34d6d2-1f8c-4802-909d-d2c9d0cbae95.png)

`sans-serif` な気がする？

Reply:
Warningのsuggestionポチってやったらserif出てきたのでそれにしてました（
あまりフォントは詳しくないので了解です！

Reply:
私もよくわからんけどGoogle Fontsがこれなのでっていうそれ←

Reply:
なるほどですゞ